### PR TITLE
Add location fields to User and Organization modules

### PR DESCRIPTION
**Pull request by zkhin**

### CODE REVIEW

File: src/components/organization/organization.resolver.ts [hunk 1]
```diff
@@ -87,6 +103,30 @@ export class OrganizationResolver {
     return true;
   }
 
+  @Mutation(() => Organization, {
+    description: 'Add a location to a organization',
+  })
+  async addLocationToOrganization(
+    @Session() session: ISession,
+    @Args('organizationId', { type: () => ID }) organizationId: string,
+    @Args('locationId', { type: () => ID }) locationId: string
```
Comment (CarsonF) [inline src/components/organization/organization.resolver.ts:112]: Could you do these like:
https://github.com/SeedCompany/cord-api-v3/blob/229348e2c1d577051ba770a49560e7c37c8a7997/src/components/authorization/authorization.resolver.ts#L34
https://github.com/SeedCompany/cord-api-v3/blob/229348e2c1d577051ba770a49560e7c37c8a7997/src/components/authorization/authorization.resolver.ts#L13-L16
That way IDs are validated.

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Review by michaelmarshall — DISMISSED** [review]

File: src/components/user/user.service.ts [hunk 2]
```diff
@@ -612,6 +619,92 @@ export class UserService {
     };
   }
 
+  async addLocation(
+    userId: string,
+    locationId: string,
+    session: ISession
+  ): Promise<void> {
+    try {
+      await this.removeLocation(userId, locationId, session);
```
Comment (CarsonF) [inline src/components/user/user.service.ts:628]: There's gotta be a better way to do this.

**Review by CarsonF — APPROVED** [review]

**Comment by CarsonF** [comment]
We've got list/add/remove locations on several resources. It could be better to abstract this in `LocationService` as just pass in the piece of the query define the relationship to the location.

**Comment by zkhin** [comment]
> We've got list/add/remove locations on several resources. It could be better to abstract this in `LocationService` as just pass in the piece of the query define the relationship to the location.

Okay, I will create a new ticket/PR for refactoring for this

**Pull request merged**